src/collective/recaptchacontactform/tests/robot/test_example.robot

```diff
@@ -129,9 +129,10 @@ I confirm that I am not a robot
 I see a message that I am a robot
   Wait until page contains element  css=.portalMessage
   Page should not contain  Thank you for your feedback
-  Page should contain  Please verify that you are not a robot
+  Page should contain  Please validate the recaptcha field before sending the form.
```

Comment:
@sneridagh I agree that the previous version was not a good choice. What about 'Please make sure the "I'm not a robot" checkbox is selected.'